pkgs/games/teeworlds/default.nix

```diff
@@ -20,21 +22,48 @@ stdenv.mkDerivation rec {
     substituteInPlace src/engine/shared/storage.cpp \
       --replace '#define DATA_DIR "data"' \
                 '#define DATA_DIR "${placeholder "out"}/share/teeworlds/data"'
+
+    # Quote nonsense is a workaround for https://github.com/NixOS/nix/issues/661
+    substituteInPlace 'other/bundle/client/Info.plist.in' \
+      --replace ${"'"}''${TARGET_CLIENT}' 'teeworlds' \
+      --replace ${"'"}''${PROJECT_VERSION}' '${version}'
+
+    # Apply https://github.com/teeworlds/teeworlds/commit/86d6687ef07f7f05457a7f67537b650656b13820
+    # Can't use fetchpatch or fetchpatch2 because of https://github.com/NixOS/nixpkgs/issues/32084
```

Comment:
I guess I can put the patch into the list of `patches`, no need to use `patch -p1`. I don't want to use `fetchurl` to fetch the patch because
> Using fetchurl for dynamically generated patches (e.g. those from GitHub) is not safe since the file can change in the future

(from https://github.com/NixOS/nixpkgs/issues/32084#issuecomment-727223713)
so I included the patch as a file instead.